src/pages/heedongChoi/Main/MainHee.scss

```
@@ -0,0 +1,274 @@
+@import url('https://fonts.googleapis.com/css2?family=Lobster&display=swap');
+@import '../styles/variables.scss';
+@import '../styles/mixin.scss';
+
+.mainWrapper {
+  min-width: 1000px;
+  background-color: #f2f2f2;
+  nav {
+    display: flex;
+    max-width: 1000px;
+    height: 5vh;
+    justify-content: space-between;
+    align-items: center;
+  }
+  .navLogo {
+    padding-left: 200px;
+    display: flex;
+    align-items: center;
+    font-family: 'Lobster', cursive;
+
+    .fa-camera-retro {
+      border-right: 1px solid black;
+      padding-right: 20px;
+    }
+    .navWestagram {
+      padding-left: 20px;
+    }
+  }
```

```suggestion
  .navLogo {
    padding-left: 200px;
    display: flex;
    align-items: center;
    font-family: 'Lobster', cursive;

    .fa-camera-retro {
      border-right: 1px solid black;
      padding-right: 20px;
    }
    
    .navWestagram {
      padding-left: 20px;
    }
  }
```
가독성을 위해 분리되는 css 속성은 띄워서 작성해주세요

---

수정 반영 했습니다!